### DISCUSSION TITLE
Add tortoise-orm to the documentation

### DISCUSSION THREAD

**Discussion by IsaiahT-Tech** [fastapi/fastapi#7346] [Questions] [answered]
### First check

* [x] I added a very descriptive title to this issue.
* [x] I used the GitHub search to find a similar issue and didn't find it.
* [x] I searched the FastAPI documentation, with the integrated search.
* [x] I already searched in Google "How to X in FastAPI" and didn't find any information.
* [x] I already read and followed all the tutorials in the docs and didn't find an answer.
* [x] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
* [x] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
* [x] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).
* [x] After submitting this, I commit to:
    * Read open issues with questions until I find 2 issues where I can help someone and add a comment to help there.
    * Or, I already hit the "watch" button in this repository to receive notifications and I commit to help at least 2 people that ask questions in the future.
    * Implement a Pull Request for a confirmed bug.

### Description

* Add a section in the docs about using FastAPI with [Tortoise ORM](https://github.com/tortoise/tortoise-orm)
* Should explain how create a basic API (Create, Read, Update, and Delete) 

### The solution you would like

A section in the docs about [Tortoise ORM](https://github.com/tortoise/tortoise-orm) should be added as it has great FastAPI integration and [Pydantic](https://github.com/samuelcolvin/pydantic) integration.

**Comment by IsaiahT-Tech** [comment]
Closed.

**Comment by tiangolo** [comment]
Thanks for coming back to close the issue. If you are still interested, you can check the official Tortoise docs: https://tortoise-orm.readthedocs.io/en/latest/examples/fastapi.html#example-fastapi

**Reply by YuriiMotov** [reply, marked as answer]
Updated link: https://tortoise.github.io/examples/fastapi.html
